R/utils.R

```diff
@@ -399,116 +399,6 @@ chunk_outputs <- function(chunks = find_chunks()$name) {
   dplyr::bind_rows(chunkoutputs)
 }
 
-#' create_xml
```

Comment:
Well I thought it made sense to move `create_xml` and the others into it's own `xml.R` the XML specific stuff was ballooning.  Although after moving the LEAVEL2_DATA_NAMES into a data object it isn't so bad.  I guess I don't have a strong opinion on moving it back.